downstream/assemblies/platform/assembly-controller-projects.adoc

```diff
@@ -1,14 +1,98 @@
+
+For each project listed, you can get the latest SCM revision image:sync.png[Refresh,15,15], edit image:leftpencil.png[Edit,15,15] the project, or copy image:copy.png[Copy,15,15] the project attributes, using the icons next to each project. 
+
+Projects can be updated while a related job is running. 
+
+In cases where you have a large project (around 10 GB), disk space on `/tmp` may be an issue.
+
+*Status* indicates the state of the project and may be one of the following (note that you can also filter your view by specific status
+types):
+
+* *Pending* - The source control update has been created, but not queued or started yet. 
+Any job (not just source control updates) stays in pending until it is ready to be run by the system. 
+Possible reasons for it not being ready are:
+
+* Because it has dependencies that are currently running so it has to wait until they are done or there is 
+* Not enough capacity to run in the locations it is configured to.
+* *Waiting* - The source control update is in the queue waiting to be executed.
+* *Running* - The source control update is currently in progress.
+* *Successful* - The last source control update for this project succeeded.
+* *Failed* - The last source control update for this project failed.
+* *Error* - The last source control update job failed to run at all. (To
+be deprecated.)
+* *Canceled* - The last source control update for the project was canceled.
+* *Never updated* - The project is configured for source control, but has never been updated.
+* *OK* - The project is not configured for source control, and is correctly in place. (To be deprecated.)
+* *Missing* - Projects are absent from the project base path of `/var/lib/awx/projects`.  
+This is applicable for manual or source control managed projects.
+
+[NOTE]
+====
+Projects of credential type `Manual` cannot update or schedule source control-based actions without being reconfigured as an SCM type
+credential.
+====
+
+//[NOTE]
+//====
+//If you delete items that are used by other work items, a message opens listing the items that are affected by the deletion and prompts you to confirm the deletion. 
+//Some screens contain items that are invalid or previously deleted, and will fail to run. 
+//
+//The following is an example of such a message:
+//+
+//image:warning-deletion-dependencies.png[Deletion dependencies]
+//====
+
+include::platform/proc-controller-adding-a-project.adoc[leveloffset=+1]
+include::platform/proc-controller-updating-a-project.adoc[leveloffset=+1]
+include::platform/ref-work-with-permissions.adoc[leveloffset=+1]
+include::platform/ref-work-with-notifications.adoc[leveloffset=+1]
+include::platform/ref-work-with-job-templates.adoc[leveloffset=+1]
+include::platform/ref-work-with-schedules.adoc[leveloffset=+1]
+include::platform/ref-projects-galaxy-support.adoc[leveloffset=+1]
+include::platform/ref-projects-collections-support.adoc[leveloffset=+1]
+endif::controller-UG[]
 
+ifdef::controller-GS[]
 [NOTE]
```

Comment:
Is this NOTE supposed to be in the User Guide?

Reply:
It's inside the controller-GS[] ifdef statement